lib/src/hydrated_bloc.dart

```diff
@@ -49,10 +45,14 @@ abstract class HydratedBloc<Event, State> extends Bloc<Event, State> {
   /// in order to keep the caches independent of each other.
   String get id => '';
 
+  /// `storageToken` is used as registration token for hydrated storage.
+  @nonVirtual
```

Comment:
Seems like the pipeline is failing because this annotation is not recognized. Is this included in the stable flutter channel?

Reply:
ok, on research now

Reply:
had to add meta.dart import